src/main/java/DukeBody/Parser.java

```diff
@@ -0,0 +1,115 @@
+package dukebody;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duketask.Deadline;
+import duketask.Event;
+import duketask.Task;
+import duketask.ToDo;
+
+
+public class Parser {
+    // members
+    private static String delimiter = " :: ";
```

Comment:
I like how you make delimiter a static var to avoid having magic string in the code! 👍